test/integration/authorization_test.go

```diff
@@ -289,10 +289,26 @@ func (test subjectAccessReviewTest) run(t *testing.T) {
 		}
 	}
 
-	if (actualResponse.Namespace != test.response.Namespace) ||
-		(actualResponse.Allowed != test.response.Allowed) ||
-		(!strings.HasPrefix(actualResponse.Reason, test.response.Reason)) {
-		t.Errorf("%s: from review\n\t%#v\nexpected\n\t%#v\ngot\n\t%#v", test.description, test.review, &test.response, actualResponse)
+	failMessage := ""
```

Comment:
hmm, any way we can share or extend `WaitForPolicyUpdate`? would be nice if this code lived in one place